README.md

```diff
@@ -1,8 +1,11 @@
 # Android Boilerplate
+<img src="https://raw.githubusercontent.com/SachseInTheHub/android-boilerplate/develop/android-ustwo.png" width="400">
```

Comment:
@Numan1617 we're just talking about how we should remove this asap since we don't like using the asset from the repo. Thanks for the suggestion buddy 👍 👍